.travis.yml

```diff
@@ -28,7 +28,7 @@ jobs:
             - clang-8
       script:
         - clang-8 -emit-llvm -c ./tests/tools/decomp/issue_4.c -o ./tests/tools/decomp/issue_4.bc
-        - docker build . -t rellic:llvm800-ubuntu18.04-amd64 -f Dockerfile --build-arg DISTRO_BASE=ubuntu18.04 --build-arg ARCH=amd64 --build-arg LLVM_VERSION=800
+        - docker build . -t rellic:llvm800-ubuntu18.04-amd64 -f Dockerfile --build-arg UBUNTU_VERSION=18.04 --build-arg ARCH=amd64 --build-arg LLVM_VERSION=800
```

Comment:
Using the build-arg `UBUNTU_VERSION` should be the preferred way now, since we use that to test how we install Python 3.7 (or later)

Reply:
No longer relevant to Python.